source/includes/steps-create-local-master-key.yaml

```diff
@@ -0,0 +1,68 @@
+title: Create the local master key
+ref: create-local-master-key
+level: 4
+content: |
+  Generate the local master key and save it to a file. You should only need to run this once:
+
+  .. tabs::
+
+     tabs:
+
+       - id: java-master-key-generator
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+
+             import java.io.FileOutputStream;
+             import java.io.IOException;
+             import java.security.SecureRandom;
+
+             public class CreateMasterKeyFile {
+               public static void main(final String[] args) {
+
+                 final byte[] localMasterKey = new byte[96];
+                 new SecureRandom().nextBytes(localMasterKey);
+
+                 String path = "master-key.txt";
+                 try (FileOutputStream stream = new FileOutputStream(path)) {
+                   stream.write(localMasterKey);
+                 } catch (IOException e)  {
+                   e.printStackTrace();
+                 }
+               }
+             }
```

Comment:
nit: should we add code comments? (I'm open to not including them if you think they're unnecessary or clutter the content.)

```java
import java.io.FileOutputStream;
import java.io.IOException;
import java.security.SecureRandom;

public class CreateMasterKeyFile {
  public static void main(final String[] args) {

    // Generate the master encryption key
    final byte[] localMasterKey = new byte[96];
    new SecureRandom().nextBytes(localMasterKey);

    // Save the key to a file on the local filesystem
    String path = "master-key.txt";
    try (FileOutputStream stream = new FileOutputStream(path)) {
      stream.write(localMasterKey);
    } catch (IOException e)  {
      e.printStackTrace();
    }
  }
}
```

Reply:
I have seen comments across the Java driver documentation of (subjectively) varying quality and need, so I thought it was acceptable and potentially helpful:
E.g. http://mongodb.github.io/mongo-java-driver/3.11/driver/tutorials/client-side-encryption/

Let me know what matches our usual convention (or how you'd like to establish it) as I do not have a strong opinion either way. Do/did we have a code documentation style guide that might help identify conventions like this?